frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/bs.ts

```diff
@@ -0,0 +1,68 @@
+export enum Providers {
+  AWS = 'AWS S3',
+  AZURE = 'Azure Blob',
+  S3 = 'S3 Compatible',
+  PVC = 'PVC',
+}
+
+export const testName = `test-${Math.random()
+  .toString(36)
+  .replace(/[^a-z]+/g, '')
+  .substr(0, 5)}-bs`;
+const inputCustomSecrets = () => {
+  const ACC_KEY = 'my_dummy_test_key';
+  const SEC_KEY = 'my_dummy_sec_key';
+  const TARGET = 'my_dummy_target';
```

Comment:
No need to declare these as constants just use it in place as these are only being used in one place.

Reply:
ack